apps/ide/src/plugins/webida.preference/view-controller.js

```diff
@@ -255,6 +244,21 @@ define([
                 preferenceDlg.onHide();
                 module.isOpened = false;
             }
+
+            // add listener for Restore default and Apply button
+            preferenceDlg.own(
```

Comment:
Thank you. That might really become serious problem.
And I think it is better to move this block to the 241-th line.